clients/java/src/test/java/io/camunda/zeebe/client/job/JobUpdateTimeoutTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.job;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.api.command.UpdateTimeoutJobCommandStep1;
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.client.api.response.UpdateTimeoutJobResponse;
+import io.camunda.zeebe.client.util.ClientTest;
+import io.camunda.zeebe.gateway.protocol.GatewayOuterClass.UpdateJobTimeoutRequest;
+import java.time.Duration;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class JobUpdateTimeoutTest extends ClientTest {
```

Comment:
🔧 We should add also some integration test, you can find some example [here](https://github.com/camunda/zeebe/tree/main/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command)

Reply:
Done and pushed, thank you